.circleci/integration/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+export SHELL=/bin/bash
+
+set -e
+
+echo "Running functional tests..."
+
```

Comment:
add a comment that says TODO since its not implemented yet

Reply:
No, it is running a basic test. We will expand it later but the basic level is implemented.